### ISSUE TITLE
Android version of App crashes after install

### ISSUE THREAD

**Issue by paulowoicho** [Chumbak/Retailstore-Attendance-Monitor#5]
After building an apk and installing it on my device, the app crashes. Did you guys have a similar issue?

**Comment by harshattray** [comment]
No, We haven't faced any issue of that sort, but this could be due to a multitude of reasons. Please check the log file generated to get a better idea of what the issue could be. 

One of the possibilities I could think of is with the `google-services.json` file that needs to be added for the firebase service to execute. This gets called initially when the app boots up and might lead to a crash in case its not added or isn't the right version.

But as i said take a look at the logfile generated , that'll give you a better picture if the localised error
